web/src/hotkey.js

```diff
@@ -294,6 +294,10 @@ export function process_escape_key(e) {
         popovers.hide_all();
         return true;
     }
+    if ($(".navbar-item:visible").is(":focus")) {
+        $(".navbar-item:visible").filter(":focus").trigger("blur");
```

Comment:
Let's extract these into any_focused and blur_focused methods called from navbar_menus module.

Also, move this block above processing text check.